packages/components/addon/components/hds/dropdown/list-item/copy-item.js

```diff
@@ -4,7 +4,8 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 
 export default class HdsDropdownListItemCopyItemComponent extends Component {
-  @tracked isSuccess = this.args.isSuccess ?? false;
+  @tracked isSuccess = false;
+  @tracked isError = false;
```

Comment:
here's my thought: or we do it properly (so we add an "error" visual state too) or we remove the `isError`.
personally I would go with removing it (we will implement the full functionality when we will build the `CopyButton` component, see #84)

Reply:
So that's interesting, I didn't know we still planned to build a copy button; I intended for the copy-item to be the final implementation. Do we have a need for a different copy button still? If there are use cases that I missed, I would like to catch them now.

Reply:
> I didn't know we still planned to build a copy button; I intended for the copy-item to be the final implementation
If you look at the Cloud UI codebase you can see that the `<ImportToTerraform>` it's using the `CopyCard` component (see https://github.com/hashicorp/cloud-ui/blob/0cb059241c392c32d9ca41d2a1293b0276fdfec0/packages/cloud-ui-core/addon/components/import-to-terraform/index.hbs#L16-L20). Now, this component is used not only in the `<ImportToTerraform>` but in may other places in Cloud UI: https://github.com/hashicorp/cloud-ui/search?q=%3CCopyCard
So we need to have a similar component, to avoid duplication of code. That's the whole reason why I started the `CopyButton` MVP: we will have that component, sooner or later (hopefully sooner).